host-interaction/gui/set-application-hook.yml

```diff
@@ -4,6 +4,12 @@ rule:
     namespace: host-interaction/gui
     author: michael.hunhoff@fireeye.com
     scope: function
+    mbc:
+      - Collection::Hooking [F0003]
+      - Credential Access::Hooking [F0003]
+      - Defense Evasion::Hooking [F0003]
+      - Persistence::Hooking [F0003]
+      - Privilege Escalation::Hooking [F0003]
```

Comment:
the hook used here is a "normal" part of implementing a GUI program, so i think we should remove these references to privesc/persistence/etc.